web_src/js/features/repo-legacy.js

```diff
@@ -436,7 +436,7 @@ async function onEditContent(event) {
       const $content = $segment;
       if (!$content.find('.dropzone-attachments').length) {
         if (data.attachments !== '') {
-          $content[0].append(data.attachments);
+          $content[0].insertAdjacentHTML('beforeend', data.attachments);
```

Comment:
As per https://github.com/go-gitea/gitea/pull/29843/files#diff-4f89ee029f289a1c40cf85fa40b0a10046a93d5fe1dac8d158e98bebd2e6e49eL439-R439 previous code had a `.dropzone-attachments` class, it it no longer needed to be created, @yardenshoham?

Reply:
It's immediately replaced../

Reply:
Oh I see the previous code created and appended it, but then immediately replaced it with the HTML string, so likely it's okay.